Lecture1/Task5/packFive/TaskFiveMain.java

```diff
@@ -0,0 +1,71 @@
+package packFive;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class TaskFiveMain {
+	public static void main(String[] args) {
+		String answer = "ДЗЯНІС ГОЛІН";
+		String mask = "****** *****";
+		char[] answerArray = answer.toUpperCase().toCharArray();
+		StringBuffer guessed = new StringBuffer(mask);
+		Set picked = new HashSet();
```

Comment:
do not use row type 
Set<String> picked = new HashSet();

Reply:
fixed